exchange/bitswap/bitswap_test.go

```diff
@@ -35,30 +34,6 @@ func TestClose(t *testing.T) {
 	bitswap.Exchange.GetBlock(context.Background(), block.Key())
 }
 
-func TestProviderForKeyButNetworkCannotFind(t *testing.T) { // TODO revisit this
```

Comment:
well it's testing what error pops out of bitswap when: (a) we have a provider, but (b) it is not reachable. we need to keep these tests. if the name is too complicated, maybe:

```
TestUnreachableProvider
```